Assets/Kottaboas/Models/Table/Wood_Plywood_Front_001_ambientOcclusion.jpg.meta

```diff
@@ -0,0 +1,92 @@
+fileFormatVersion: 2
```

Comment:
do you have licenses you can include with any models / textures that you got from somewhere else?